libs/commons/Logs_helpers.ml

```diff
@@ -104,3 +104,18 @@ let setup_logging ~force_color ~level =
          (* those are the one we are really interested in *)
          | "application" -> ()
          | s -> failwith ("Logs library not handled: " ^ s))
+
+let with_err_tag ?(tag = " ERROR ") () =
+  ANSITerminal.sprintf
+    [ ANSITerminal.white; ANSITerminal.Bold; ANSITerminal.on_red ]
+    "%s" tag
+
+let with_warn_tag ?(tag = " WARN ") () =
```

Comment:
In theory we should not expose those functions.
Logs.err, Logs.warn, should do the right thing and use the right color with the right prefix.
Maybe we could add a Logs_helpers.success for the success thing, but otherwise
all errors, warnings should use the same consistent output.

Reply:
While I agree here for the long-term, this implementation was +1 "style points" over using a hardcoded escape sequence copied throughout the Login and Logout subcommands (e.g. https://github.com/returntocorp/semgrep/blob/develop/src/osemgrep/cli_scan/Scan_subcommand.ml#L380)

I saw that we do have a `logger#error` utility used for the main language library code, but I wasn't comfortable enough with OCaml to create my own logger for the CLI commands.

Perhaps I can add a TODO here?

Reply:
Just add a TODO